person.rb

```diff
@@ -1,15 +1,15 @@
-require_relative 'nameable'
+require_relative 'rental'
 
-class Person < Nameable
+class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] For the Person class, Getters and Setters for @rentals should be present according to the project requirement. Let's have something like this `attr_accessor :name, :age, :rentals` to achieve that. 👍🏽